src/azure-cli/azure/cli/command_modules/role/_params.py

```diff
@@ -120,7 +120,7 @@ def load_arguments(self, _):
         c.argument('identifier_uri', help='graph application identifier, must be in uri format')
         c.argument('spn', help='service principal name')
         c.argument('upn', help='user principal name, e.g. john.doe@contoso.com')
-        c.argument('query_filter', options_list=['--filter'], help='OData filter')
+        c.argument('query_filter', options_list=['--filter'], help='OData filter, e.g. displayname eq \'test\' and servicePrincipalType eq \'Application\'')
```

Comment:
Also. We may put the link here for more information: https://docs.microsoft.com/en-us/previous-versions/azure/ad/graph/howto/azure-ad-graph-api-supported-queries-filters-and-paging-options

Reply:
This page is out-of-date so that I prefer not to mention this link here in case it is not accessible one day

Reply:
Makes sense! Maybe we can improve this during MS Graph integration.